--- a/home/analytics/venvs/pipeline/local/lib/python2.7/site-packages/luigi/contrib/mysqldb.orig.py
+++ b/home/analytics/venvs/pipeline/local/lib/python2.7/site-packages/luigi/contrib/mysqldb.py
@@ -96,6 +96,10 @@ class MySqlTarget(luigi.Target):
         Using a separate connection since the transaction might have to be reset"""
         connection = self.connect(autocommit=True)
         cursor = connection.cursor()
+        # Fix a bug where MySQL will hang forever "Waiting for table metadata lock" if this table exists
+        cursor.execute("SELECT COUNT(*) FROM information_schema.tables WHERE table_name = '{}'".format(self.marker_table))
+        if cursor.fetchone()[0] == 1:
+            return connection.close()
         try:
             cursor.execute(
                 """ CREATE TABLE {marker_table} (