--- a/home/analytics/venvs/pipeline/local/lib/python2.7/site-packages/luigi/hdfs.orig.py
+++ b/home/analytics/venvs/pipeline/local/lib/python2.7/site-packages/luigi/hdfs.py
@@ -78,9 +78,9 @@ def tmppath(path=None):
 
     #2. Figure out what to place
     if path is not None:
-        if path.startswith(temp_dir + '/'):
+        if path.startswith(base_dir + '/'):
             #Not 100%, but some protection from directories like /tmp/tmp/file
-            subdir = path[len(temp_dir):]
+            subdir = path[len(base_dir):]
         else:
             #Protection from /tmp/hdfs:/dir/file
             parsed = urlparse.urlparse(path)